test/integration_test.go

```diff
@@ -202,7 +204,8 @@ type AssertMapOutput struct {
 func runTest(t *testing.T, startBlock int64, exclusiveEndBlock uint64, moduleNames []string, newBlockGenerator NewTestBlockGenerator) (moduleOutputs []string) {
 	//_, _ = logging.ApplicationLogger("test", "test")
 
-	err := os.RemoveAll("/tmp/test.store")
+	filepath := filepath.Join(os.TempDir(), "test.store")
+	err := os.RemoveAll(filepath)
```

Comment:
And there is no need to remove the file anymore, `t.TempDir` is automatically cleaned up when the test (and its sub-tests) end

Reply:
Done